File: samples/server/petstore/jaxrs-cxf-annotated-base-path/src/main/java/io/swagger/api/impl/PetApiServiceImpl.java
package io.swagger.api.impl;

import io.swagger.api.*;
import java.io.File;
import io.swagger.model.ModelApiResponse;
import io.swagger.model.Pet;

import java.io.InputStream;
import java.io.OutputStream;
import java.util.List;
import java.util.Map;
import javax.ws.rs.*;
import javax.ws.rs.core.Response;
import org.apache.cxf.jaxrs.model.wadl.Description;
import org.apache.cxf.jaxrs.model.wadl.DocTarget;

import org.apache.cxf.jaxrs.ext.multipart.*;

import io.swagger.annotations.Api;

public class PetApiServiceImpl implements PetApi {
    public void addPet(Pet body) {
        // TODO: Implement...
        
        
    }
    
    public void deletePet(Long petId, String apiKey) {
        // TODO: Implement...
        
        
    }
    
    public List<Pet> findPetsByStatus(List<String> status) {
        // TODO: Implement...
        
        return null;
    }
    
    public List<Pet> findPetsByTags(List<String> tags) {
        // TODO: Implement...
        
        return null;
    }
    
    public Pet getPetById(Long petId) {
        // TODO: Implement...
        
        return null;
    }
    
    public void updatePet(Pet body) {
        // TODO: Implement...
        
        
    }
    
    public void updatePetWithForm(Long petId, String name, String status) {
        // TODO: Implement...
        
        
    }
    
    public ModelApiResponse uploadFile(Long petId, String additionalMetadata,  Attachment fileDetail) {
        // TODO: Implement...
        
        return null;
    }
    
}

File: samples/server/petstore/kotlin-springboot-responseentity/src/main/kotlin/org/openapitools/api/PetApiServiceImpl.kt
package org.openapitools.api

import org.openapitools.model.ModelApiResponse
import org.openapitools.model.Pet
import org.springframework.http.ResponseEntity
import org.springframework.stereotype.Service


@Service
class PetApiServiceImpl : PetApiService {

    override fun addPet(body: Pet): ResponseEntity<Unit> {
        TODO("Implement me")
    }

    override fun deletePet(petId: kotlin.Long, apiKey: kotlin.String?): ResponseEntity<Unit> {
        TODO("Implement me")
    }

    override fun findPetsByStatus(status: kotlin.collections.List<kotlin.String>): ResponseEntity<List<Pet>> {
        TODO("Implement me")
    }

    override fun findPetsByTags(tags: kotlin.collections.List<kotlin.String>): ResponseEntity<List<Pet>> {
        TODO("Implement me")
    }

    override fun getPetById(petId: kotlin.Long): ResponseEntity<Pet> {
        TODO("Implement me")
    }

    override fun updatePet(body: Pet): ResponseEntity<Unit> {
        TODO("Implement me")
    }

    override fun updatePetWithForm(petId: kotlin.Long, name: kotlin.String?, status: kotlin.String?): ResponseEntity<Unit> {
        TODO("Implement me")
    }

    override fun uploadFile(petId: kotlin.Long, additionalMetadata: kotlin.String?, file: org.springframework.core.io.Resource?): ResponseEntity<ModelApiResponse> {
        TODO("Implement me")
    }
}
